packages/runtime-common/create-response.ts

```diff
@@ -9,6 +12,8 @@ export function createResponse(
     headers: {
       ...init?.headers,
       'X-Boxel-Realm-Url': unresolvedRealmURL,
+      ...(permissions['*'].includes('read') && { 'X-Realm-Public-Readable': 'true' }),
+      ...(permissions['*'].includes('write') && { 'X-Realm-Public-Writable': 'true' }),
```

Comment:
Since we already have `X-Boxel-Realm-…` maybe these should follow that pattern? `X-Boxel-Realm-Public-Readable`

Reply:
Agreed 👍